tests/test_dataUnit.py

```diff
@@ -24,15 +24,14 @@
 import lsst.utils
 import lsst.utils.tests
 
-from lsst.daf.butler.core.schema import SchemaConfig
-from lsst.daf.butler.core.dataUnit import DataUnit, DataUnitRegistry
+from lsst.daf.butler.core.dataUnit import DataUnit, DataUnitRegistryConfig, DataUnitRegistry
```

Comment:
They should be indeed. But there may be cases where implementation details (instead of API) are unit tested, and in those cases I'd like to not see them exported from `core`.

Reply:
Why?  That just seems like testing multiple orthogonal implementation details (where the code lives, as well as what it does) simultaneously.

Reply:
If it's available through `lsst.daf.butler` we should use it. `lsst.daf.butler.core.utils` is not exported since it's internal support tools, but those are needed for tests.